package.json

```diff
@@ -48,6 +48,7 @@
     "@types/mocha": "^8.2.1",
     "@types/node": "^14.14.31",
     "@types/sinon": "^9.0.10",
+    "@types/sinonjs__fake-timers": "^8.1.5",
```

Comment:
This fixes the build step since I had this error while building:
```js
> @twilio/video-processors@2.2.1-dev build:version
> node ./scripts/version.js

error TS2688: Cannot find type definition file for 'sinonjs__fake-timers'.
  The file is in the program because:
    Entry point for implicit type library 'sinonjs__fake-timers'


Found 1 error.

ERROR: "build:es5" exited with 2.
```